codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -135,6 +134,24 @@ public void onPriorityTreeParentChanging(Http2Stream stream, Http2Stream newPare
         });
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * Any queued {@link FlowControlled} objects will be sent.
+     */
+    @Override
+    public void channelHandlerContext(ChannelHandlerContext ctx) throws Http2Exception {
+        this.ctx = checkNotNull(ctx, "ctx");
```

Comment:
Good point. It is expected to be called 1 time on `handlerAdded` but I should investigate clean up conditions and behavior.